docs/content/comparisons/druid-vs-commericial-solutions.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: doc_page
+---
+
+Druid vs Proprietary Commercial Solutions (Vertica/Redshift)
```

Comment:
And for this specific case, I don't think it makes sense to combine Vertica with a SaaS like Redshift. We could also just drop the Vertica comparison, as it is kind of bare bones and not really that useful.